templates/taxi/car_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <p>Are you sure you want to delete the car: {{ car.model }}?</p>
+	<form method="post" action="">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+    <a class="btn btn-primary" href="{{ car.get_absolute_url }}">Cancel</a>
```

Comment:
You can use smth like that

`<a href="{% url 'car-detail' pk=car.pk %}"....`

So, no need to create external method to get link